### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by TymanWasTaken**
```md
# Changes
- Added GitHub workflow that auto uploads compiled files
- Added a readme
- Updated mod version in the files (stop just renaming the files moul >:c)
- Made version variable in mod get replaced by version in build.gradle at compile time (totally not copied from cowlection)
- Fixed forge maven url
- Fixed quote consistency in build.gradle
- Fixed indentation consistency in build.gradle
- Updated gradle
- Changed to asbyth's fork of forgegradle
- Changed to xcfrg's fork of mixingradle (normal mixingradle doesn't work with updated gradle version)
- Updated shadow plugin
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,7 +44,7 @@ repositories {
 }
 
 dependencies {
-    compile("org.spongepowered:mixin:0.7.11-SNAPSHOT")
+    implementation("org.spongepowered:mixin:0.7.11-SNAPSHOT")
```
Comment (xcfrg) [inline build.gradle]: add
```groovy
annotationProcessor("org.spongepowered:mixin:0.7.11-SNAPSHOT")
```

Comment (xcfrg) [inline build.gradle]: file size could be reduced by excluding modules like guava gson, launchwrapper, commons-io

File: build.gradle [hunk 2]
```diff
@@ -3,11 +3,12 @@ buildscript {
         jcenter()
```
Comment (xcfrg) [inline build.gradle]: jcenter will be shutdown soon, best to remove it now

**Review by xcfrg — COMMENTED** [review]
Also use stable_22

**Comment by TymanWasTaken** [comment]
> Also use stable_22

@xcfrg I tried it yelled at me with code that is obfuscated so I just reverted it

File: build.gradle (hunk 1 above)
Comment (TymanWasTaken) [reply build.gradle]: add after implementation?

Comment (TymanWasTaken) [reply build.gradle]: im going to assume yes

**Comment by proudmuslim-dev** [comment]
Squash your commits please, thanks

**Comment by TymanWasTaken** [comment]
Im just going to close this because its not getting merged anyways :sadge:

**Comment by Scherso** [comment]
> ```lisp
> # Changes
> - Added GitHub workflow that auto uploads compiled files
> - Added a readme
> - Updated mod version in the files (stop just renaming the files moul >:c)
> - Made version variable in mod get replaced by version in build.gradle at compile time (totally not copied from cowlection)
> - Fixed forge maven url
> - Fixed quote consistency in build.gradle
> - Fixed indentation consistency in build.gradle
> - Updated gradle
> - Changed to asbyth's fork of forgegradle
> - Changed to xcfrg's fork of mixingradle (normal mixingradle doesn't work with updated gradle version)
> - Updated shadow plugin
> ```

Wonder if it works with hytilities, i’ll try it when i’m home

**Pull request closed without merging**